### PR TITLE
Add python package muler to conda-forge starting at version 0.2.4

### PR DESCRIPTION
**Pull request by gully**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gully** [comment]
@conda-forge/help-python ready for review :smiley:   :pray:

File: recipes/muler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (dopplershift) [inline recipes/muler/meta.yaml:5]:
```suggestion
```

File: recipes/muler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  sha256: ee50e30aa955d1e6a033539eb28d1dab768b2b40d32b4659eab4eb5294964a01
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/muler/meta.yaml:24]:
```suggestion
    - python >=3.6
```

File: recipes/muler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - pip
+  run:
+    - python >=3.6
+
```
Comment (dopplershift) [inline recipes/muler/meta.yaml:28]:
```suggestion
```

File: recipes/muler/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - astropy >=4.1
+    - specutils >=1.2
+    - numpy 
+    - scipy 
+    - celerite2
+    - pandas
+    - matplotlib
```
Comment (dopplershift) [inline recipes/muler/meta.yaml:35]: Unless you need a PyQt-based GUI, better to depend on matplotlib-base:
```suggestion
    - matplotlib-base
```

File: recipes/muler/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "muler" %}
+{% set version = "0.2.1" %}
```
Comment (dopplershift) [inline recipes/muler/meta.yaml:7]: Latest is now 0.2.4. It'd be good to go ahead and updated unless you specifically need 0.2.1.

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Comment by gully** [comment]
Ok, I made all the changes, thank you so much @dopplershift (nice handle btw).  Awesome service here! 😄

**Comment by gully** [comment]
@conda-forge/help-python ready for review

**Comment by synapticarbors** [comment]
I opened a PR upstream to get the license file included in the source distribution. In subsequent releases of the package, please check to see if the manually packaged copy can be removed:

https://github.com/OttoStruve/muler/pull/59

**Pull request merged**